util/templates/generate/main.go

```diff
@@ -13,16 +13,26 @@ import (
 )
 
 const (
-	language    = "de"
 	docsPath    = "../../../templates/docs"
 	websitePath = "../../../templates/evcc.io"
 )
 
 //go:generate go run main.go
 
 func main() {
+
+	for _, lang := range []string{"de", "en"} {
+		generateDocs(lang)
+	}
+
+	if err := generateBrandJSON(); err != nil {
+		panic(err)
+	}
+}
+
+func generateDocs(lang string) {
 	for _, class := range []templates.Class{templates.Meter, templates.Charger, templates.Vehicle} {
-		path := fmt.Sprintf("%s/%s", docsPath, class)
+		path := fmt.Sprintf("%s/%s/%s", docsPath, lang, strings.ToLower(class.String()))
```

Comment:
Mhhm. Warum ging das vorher mit der class? Wg. existierender Ordner ignoriert?

Reply:
Ja, das hat vorher auf case-sensitiven Dateisystemen nur funktioniert weil die Class-Ordner immer existierten. Auf meinem Mac lief das auch ohne diese Änderung aber in den Actions fiel das auf die Nase weil lesen und schreiben anderen Case hatten.